etc/kubernetes/helm/Chart.yaml

```diff
@@ -0,0 +1,3 @@
+name: enterprise-gateway
+version: 1.0.0
```

Comment:
I'm thinking we would want the Chart version at 1.0 for the GA release, but perhaps have it prior to that until we GA.  Something like `0.8.0` or `0.9.0` perhaps?

Reply:
Container/Kubernetes support is only available with EG 2.x release, so I would recommend aligning the first release of helm with that to avoid confusion. But I am flexible.

Reply:
@kevin-bates I've set the version number to `0.8.0` as per your suggestion.

@lresende I may be misunderstanding, but given that this PR would merge into master, the Helm support wouldn't actually make its way into general availability until the next official release (or unless someone clones master directly).